### DISCUSSION TITLE
有什么途径可以参与开发？

### DISCUSSION THREAD

**Discussion by biaov2017** [biaov/MINE-H5-UI#41] [Q&A] [answered]
我发现这个框架还有许多的不足之处，想参与开发

**Comment by biaov** [comment, marked as answer]
`2023-06-19`

- 可以 fork
- 然后修改不足之处
- 代码风格遵循 eslint + prettier
- 然后创建新分支修改
- 修改完成之后提 PR 即可

**Reply by biaov2017** [reply]
OK
